docker-maven-plugin/src/main/java/org/openmrs/maven/plugins/AbstractDockerMojo.java

```diff
@@ -21,10 +21,10 @@
 
 abstract class AbstractDockerMojo extends AbstractMojo {
 
-    protected static final String DEFAULT_MYSQL_CONTAINER = "openmrs-sdk-mysql-v3-2";
+    protected static final String DEFAULT_MYSQL_CONTAINER = "openmrs-sdk-mysql-v3-3";
```

Comment:
Why isn't this named `openmrs-sdk-mysql-v8-4-1`?

Reply:
Since we named it 'openmrs-sdk-mysql-v3-2' while using MySQL 5.6, I assumed it wasn't referencing the MySQL version.

Reply:
So what does `v3-3` mean?

Reply:
New openmrs-sdk container version. That's what I thought. I think it would be better to rename it with the MySQL version.

Reply:
> I think it would be better to rename it with the MySQL version.

Oh ya man! 👍